docs/source/common/lightning_module.rst

```diff
@@ -1195,6 +1195,7 @@ for more information.
             on_after_backward()
 
             on_before_optimizer_step()
+            configure_gradient_clipping()
```

Comment:
good that you added it here. because we discover here something not so nice. 

the `on_before_optimizer_step()` is now not running anymore directly before the optimizer step. do we want honor that and switch it up?

Reply:
EDIT: I see it may not be so easy to do it here. 
But still, I think it's slightly problematic

Reply:
yes.. i noticed.. so didn't change the sequence here.. maybe in another PR.